dap-mode.el

```diff
@@ -1598,6 +1621,49 @@ point is set."
   (interactive)
   (dap-mode t))
 
+
+;; Auto configure
+
+(defun dap--show-feature-window (_session)
+  "Show auto confiured feature windows."
+  (seq-doseq (feature-start-stop dap-auto-configure-features)
+    (when-let ((start-stop (alist-get feature-start-stop dap-feature->start-stop)))
+      (funcall (car start-stop)))))
+
+(defun dap--hide-feature-window (_session)
+  "Hide all debug windows when sessions are dead."
+  (unless (-filter 'dap--session-running (dap--get-sessions))
+    (seq-doseq (feature-start-stop dap-auto-configure-features)
+      (-when-let* ((feature-start-stop (alist-get feature-start-stop dap-feature->start-stop))
+                   (buffer-name (symbol-value (cdr feature-start-stop))))
```

Comment:
If this one is meant for buffer name, why don't you assign the buffer name in the `dap-feature->start-stop` as string instead of symbol?

Reply:
How? The problem is that `dap-feature->start-stop` returns for example `dap-ui--breakpoints-buffer` instead of the value `"*dap-ui-breakpoints*"`, is there any better way to already get the value from the `dap-feature->start-stop`?

Reply:
Ah, I see, I misread `symbol-value` to `symbol-name` 😄